.travis.yml

```diff
@@ -21,7 +21,7 @@ jobs:
   include:
     - stage: linting
       name: "pre-commit"
-      install: pip install pre-commit
+      install: pip install pre-commit python-magic
```

Comment:
@vancouver29 I'm wondering why adding this ?

Reply:
@rousseldenis after executing `pre-commit` i got the error that `magic` is not available. Hence, that is why I added `python-magic` there

Reply:
Where is it required ?

Reply:
Ok, see, looked into files headers.

You should add an external_dependencies to that library in module manifest instead.

Reply:
@rousseldenis I tried to add either `magic` or `python-magic` to `external_dependencies` in the manifest, but the test seems to be failed and it does not recognize this dependency. Hence, i think the only way to make it work here is to add `python-magic` into the `.travis.yml` file. Or do you have any idea?

Reply:
> Or do you have any idea?

Yes. The good one is to switch to Github Actions and then, setuptools-odoo will detect the entry in external_dependencies.

I'll do the PR for that.

Reply:
@vancouver29 #359

Reply:
@rousseldenis Great :)! I'm looking forward that your PR being merged, then I can finish this PR and start the migration of this module for V14 & V15